web/package.json

```diff
@@ -30,6 +30,7 @@
     "typecheck": "tsc --noemit",
     "typecheck:cypress": "tsc --noemit --project cypress/tsconfig.json",
     "webflow:fundings-sync": "node ./src/scripts/webflow/fundingSync.mjs --sync",
+    "webflow:fundings-sync-full": "node ./src/scripts/webflow/fundingSync.mjs --full",
```

Comment:
We have a sector sync script that doesn't appear to be used, but is necessary for the funding sync to work. Added for ease of access while testing this locally. Left in because we will probably want to make sure to sync our sectors if they ever change.